test/e2e/scheduling/limit_range.go

```diff
@@ -236,7 +236,7 @@ var _ = SIGDescribe("LimitRange", func() {
 		the limitRange by collection with a labelSelector it MUST delete only one
 		limitRange.
 	*/
-	framework.ConformanceIt("should list, patch and delete a LimitRange by collection", func(ctx context.Context) {
+	framework.ConformanceIt("should list, patch and delete a LimitRange by collection", ginkgo.SpecTimeout(wait.ForeverTestTimeout), func(ctx context.Context) {
```

Comment:
how does this interact with the helpers that we have on the `framework.BeforeEach`, like these wait

https://github.com/kubernetes/kubernetes/blob/00aae4c10c204d258e62a96f30970e9b91b29149/test/e2e/framework/framework.go#L246-L250

is `SpecTimeout` taking into account those times? if affirmative this is changing the behavior and we should be cautious

Reply:
You are right. To get the same semantic as before (= timeout applies to code inside the `It`) one has to use `NodeTimeout`.

Will change it.

/lgtm cancel